dll/win32/devmgr/devmgmt/MainWindow.cpp

```diff
@@ -652,11 +651,14 @@ CDeviceManager::OnCommand(_In_ WPARAM wParam,
 
         case IDM_ABOUT:
         {
-            // Apportion blame
-            MessageBoxW(m_hMainWnd,
-                        L"ReactOS Device Manager\r\nCopyright Ged Murphy 2015",
-                        L"About",
-                        MB_OK | MB_APPLMODAL);
+            CAtlStringW szAppName;
```

Comment:
No error checking?
Looks like `szAppAuthors` can be null, and `DestroyIcon` can handle a null ptr. Didn't look into `szAppName`

Reply:
szAppName is the only pointer which can not be NULL for ShellAboutW.
However, I understand that CAtlStringW is "safe" with regard to this case. Am I wrong ?

Reply:
I wouldn't say `DestroyIcon` can be null, it just so happens that if you drill into win32k it has some protection there. There's nothing in the docs to suggest you can pass null.
Regarding `CAtlStringW` being safe, it's unrelated. An empty string will be null, so you're passing null into the API. 

You should really be null checking `szAppName` and `hIcon` for null to avoid crashing the app.

Reply:
OK for checking hIcon for safe `DestroyIcon`.
For szAppName, I understand that `CAtlStringW `would produce a null string (L"\0"), but not a null pointer. But i'll add robustness case for that as well.

Reply:
Oh it does? It's been so long since I worked with atl...
The fix looks much better now anyway 👍

Reply:
Anyway, thank you very much for this quick and efficient review and feedback !
Happy to update and complement the very good work you did in the past on this module

Reply:
I kind of remember @ThFabba telling me code should not include defaults for resources...
Does that not apply here?

Reply:
Sorry i've not seen any explicit design rule for that and @gedmurphy asked me to change.